src/main/java/filter/Parser.java

```diff
@@ -106,26 +106,22 @@ private FilterExpression parseExpression(int precedence) {
         return left;
     }
 
-    private FilterExpression parseQuotedKeywords(Token token) {
-        FilterExpression result = parseKeywords();
+    private FilterExpression parseQuotedKeywords() {
+        FilterExpression result = parseQuotedContent(QualifierType.KEYWORD);
         consume(TokenType.QUOTE);
         return result;
     }
 
-    private FilterExpression parseKeywords() {
-        return parseKeywords("keyword");
-    }
-
-    private FilterExpression parseKeywords(String qualifierName) {
+    private FilterExpression parseQuotedContent(QualifierType type) {
```

Comment:
I am not sure about the name. I feel that `content` means that it (the input) has been parsed, so we do not need to so-called parse content in this scenario - the `parseContent` can be used, for example, in resolving user alias or the `i:o` case in discussion. I can't come up with a better name, though. What do you think?